<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::GroupedDragging</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,3,5,3,5,3,5,3,5,3,5" columnstretch="2,3,2">
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="1">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #2E3B4E; /* Темно-синий цвет фона */
    color: white; /* Белый цвет текста */
    border: 2px solid #1A2C40; /* Более тёмная граница */
    border-radius: 10%; /* Скруглённые края */
    padding: 2px 12px; /* Отступы вокруг текста */
    
    font-size: 16px; /* Размер шрифта */
    font-weight: bold; /* Жирный текст */
    text-align: center;
}

QPushButton:hover {
    background-color: #3A4A62; /* Светло-синий при наведении */
    border-color: #293E58; /* Цвет границы при наведении */
}

QPushButton:pressed {
    background-color: #1C2535; /* Темный цвет фона при нажатии */
    border-color: #1A2C40; /* Граница остается темной */
}
</string>
      </property>
      <property name="text">
       <string>Загрузить</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Bauhaus 93</family>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:700; color:#ffffff;&quot;&gt;Главное Меню&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="pushButton_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #2E3B4E; /* Темно-синий цвет фона */
    color: white; /* Белый цвет текста */
    border: 2px solid #1A2C40; /* Более тёмная граница */
    border-radius: 10%; /* Скруглённые края */
    padding: 2px 12px; /* Отступы вокруг текста */
    
    font-size: 16px; /* Размер шрифта */
    font-weight: bold; /* Жирный текст */
    text-align: center;
}

QPushButton:hover {
    background-color: #3A4A62; /* Светло-синий при наведении */
    border-color: #293E58; /* Цвет границы при наведении */
}

QPushButton:pressed {
    background-color: #1C2535; /* Темный цвет фона при нажатии */
    border-color: #1A2C40; /* Граница остается темной */
}
</string>
      </property>
      <property name="text">
       <string>Об авторах</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #2E3B4E; /* Темно-синий цвет фона */
    color: white; /* Белый цвет текста */
    border: 2px solid #1A2C40; /* Более тёмная граница */
    border-radius: 10%; /* Скруглённые края */
    padding: 2px 12px; /* Отступы вокруг текста */
    
    font-size: 16px; /* Размер шрифта */
    font-weight: bold; /* Жирный текст */
    text-align: center;
}

QPushButton:hover {
    background-color: #3A4A62; /* Светло-синий при наведении */
    border-color: #293E58; /* Цвет границы при наведении */
}

QPushButton:pressed {
    background-color: #1C2535; /* Темный цвет фона при нажатии */
    border-color: #1A2C40; /* Граница остается темной */
}
</string>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #2E3B4E; /* Темно-синий цвет фона */
    color: white; /* Белый цвет текста */
    border: 2px solid #1A2C40; /* Более тёмная граница */
    border-radius: 10%; /* Скруглённые края */
    padding: 2px 12px; /* Отступы вокруг текста */
    
    font-size: 16px; /* Размер шрифта */
    font-weight: bold; /* Жирный текст */
    text-align: center;
}

QPushButton:hover {
    background-color: #3A4A62; /* Светло-синий при наведении */
    border-color: #293E58; /* Цвет границы при наведении */
}

QPushButton:pressed {
    background-color: #1C2535; /* Темный цвет фона при нажатии */
    border-color: #1A2C40; /* Граница остается темной */
}
</string>
      </property>
      <property name="text">
       <string>Новая игра</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="1">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
